### ISSUE TITLE
Very specifc .js injection issue

### ISSUE THREAD

**Issue by outdoorsdev** [1fexd/600-sound-volume#3]
I run a wordpress website and when I'm editing rich text fields in the post editor I can see in the text field that the following is added to the text. It's not present in the visual editor.

It looks like the script is adding this js. I'm not sure if this is very dodgy practice left over from the original plugin, or unintended insertion of the code for the new version of the add-on.

```
&nbsp;

<script src="moz-extension://e3c30c3a-0d5a-4597-9353-5eb708532b37/js/app.js" type="text/javascript"></script>
```

That means when I don't check the text, this js gets pulled into the site. It doesn't really make a difference as it's trying to load a file locally, but still odd behaviour.

**Comment by 1fexd** [comment]
Hey there, sorry for getting back to you this late. Would it be possible for you to link your website?

**Comment by DarkOverord** [comment]
Firefox 128.0.3 (64-bit), Win10

Adding in on this, almost a year later, same issue - It's inserting a HTML script that calls for the app.js in the extension's UUID in to WordPress' WYSIWYG editor. The old one does this too (which me looking in to it is how I found out about the privacy issue, thanks for this!) 

Specifically what happens is that script only shows up when you hit save draft/update/whatever for that WYSIWYG editor so I guess it's capturing something at that moment of page refresh? It doesn't do it if you've got the plain text ver open, only the WYSIWYG editor. I've not been able to recreate this in other editors yet, but I don't really encounter other editors that nab HTML like WP does!

There's a couple reviews on the original extension that also bring up this issue so it's absolutely a holdover from that [1](https://addons.mozilla.org/en-US/firefox/addon/600-sound-volume/reviews/2099261/), [2](https://addons.mozilla.org/en-US/firefox/addon/600-sound-volume/reviews/2039812/)

**Comment by 1fexd** [comment]
Should be fixed via 21f69f1cb9ac155025177803691b86e56d54461c, awaiting approval on the addons store

**Comment by DarkOverord** [comment]
Can confirm so far I've not seen it happen since the update rolled out!
